MapboxAndroidDemo/src/main/res/values/activity_strings.xml

```diff
@@ -0,0 +1,208 @@
+    <string name="directions_activity_marker_options_origin_title">Origin</string>
+    <string name="directions_activity_marker_options_origin_snippet">Alhambra</string>
+    <string name="directions_activity_marker_options_destination_title">Destination</string>
+    <string name="directions_activity_marker_options_destination_snippet">Plaza del Triunfo</string>
+    <string name="directions_activity_toast_message">Route is %1$f meters long.</string>
+
+    <!--Strings for OfflineManager activity-->
+    <string name="dialog_title">Name new region</string>
+    <string name="dialog_message">Downloads the map region you currently are viewing</string>
+    <string name="dialog_positive_button">Download</string>
+    <string name="dialog_negative_button">Cancel</string>
+    <string name="dialog_toast">Region name cannot be empty.</string>
+    <string name="end_progress_success">Region downloaded successfully.</string>
+    <string name="toast_no_regions_yet">You have no regions yet.</string>
+    <string name="toast_region_deleted">Region deleted</string>
+    <string name="navigate_positive_button">Navigate to</string>
+    <string name="navigate_neutral_button_title">Delete</string>
+    <string name="navigate_negative_button_title">Cancel</string>
+    <string name="navigate_title">List</string>
+    <string name="region_name">Region %1$d</string>
+    <string name="set_region_name_hint">Enter name</string>
+
+    <!--SimpleOfflineMapActivity activity-->
+    <string name="simple_offline_end_progress_success">Region downloaded successfully.</string>
+
+    <!--FeatureCountActivity activity-->
+    <string name="feature_count_snackbar_feature_size">%1$d features in box</string>
+
+    <!--QueryFeatureActivity activity-->
+    <string name="query_feature_marker_title">Properties:</string>
+    <string name="query_feature_marker_snippet">No feature properties found</string>
+    <string name="query_feature_no_properties_found">No feature properties found</string>
+
+    <!--LocationPickerActivity activity-->
+    <string name="location_picker_select_location_button_cancel">Cancel</string>
+    <string name="location_picker_select_location_button_select">Select a location</string>
+    <string name="location_picker_dropped_marker_snippet_no_results">No results</string>
+
+    <!--LosAngelesTourismActivity activity-->
+    <string name="fab_title_hotels">Hotels</string>
+    <string name="fab_title_parks">Parks</string>
+    <string name="fab_title_attractions">Attractions</string>
+
+    <!--OffRouteActivity activity-->
+    <string name="off_route_toast_press_map">Press map to add destination</string>
+    <string name="toast_no_routes_found">No routes found</string>
+    <string name="off_route_toast_off_route">Off route</string>
+
+    <!--LanguageSwitchActivity activity-->
+    <string name="language_switch_options_menu_language_english">English</string>
+    <string name="language_switch_options_menu_language_french">Français</string>
```

Comment:
Either these language names should be in English, or they should be marked as untranslatable. Otherwise, they’ll say e.g. “Français” in the English and French localizations but “Francés” in the Spanish localization.